engine/release-notes/23.0.md

```diff
@@ -41,6 +41,37 @@ Changing the version format is a stepping-stone towards Go module compatibility,
 but the repository doesn't yet use Go modules, and still requires using a "+incompatible" version.
 Work continues towards Go module compatibility in a future release.
 
+## 23.0.3
+
+{% include release-date.html date="2023-04-04" %}
+
+> **Note**
+> 
+> Due to an issue with CentOS 9 Stream's package repositories, packages for
+> CentOS 9 are currently unavailable. Packages for CentOS 9 may be added later,
+> or as part of the next (23.0.4) patch release.
```

Comment:
I added a note about the missing packages for CentOS 9

Reply:
Yup, and runc as well; I pasted both errors in a PR there; https://github.com/opencontainers/runc/pull/3813#issuecomment-1497107025